src/components/vm/nics/vmNicsCard.jsx

```diff
@@ -105,6 +105,55 @@ VmNetworkActions.propTypes = {
     networks: PropTypes.array.isRequired,
 };
 
+const NetworkSource = ({ network, networkId, vm, hostDevices }) => {
+    const id = vmId(vm.name);
+    const checkDeviceAviability = (network) => {
```

Comment:
"Aviability" means checking if the device can fly, right? ✈️

The typo already exists in the moved code, it's just a funny typo.